app/scripts/modules/core/application/config/serializeApplicationSection.directive.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.application.config.serialize.directive', [
+    require('angular-ui-router'),
+    require('./../../serialize/serialize.write.service.js'),
+    require('../../confirmationModal/confirmationModal.service.js'),
+    require('../../overrideRegistry/override.registry.js'),
+  ])
+  .directive('serializeApplicationSection', function (overrideRegistry) {
+    return {
+      restrict: 'E',
+      templateUrl: overrideRegistry.getTemplate('serializeApplicationSectionDirective', require('./serializeApplicationSection.directive.html')),
```

Comment:
This will probably confuse people, since it implies that there's an override somewhere. You can just `require` your template.